### PR TITLE
Adding a new article on better code quality

### PR DESCRIPTION
**Pull request by ivanleomk**
I wrote up a draft of an article which showcases how to improve your python code. I tried to rewrite a lot of it so that it reads like a mini reflection.

Testing a potential hook of - hey i am a frontend dev, now iw rite more python. what changed for me.

### CODE REVIEW

File: posts/from-javascript-to-python.mdx [hunk 1]
```diff
@@ -0,0 +1,249 @@
+---
+title: From Javascript to Python
```
Comment (jxnl) [inline posts/from-javascript-to-python.mdx:2]: title could be stronger  for SEO

File: posts/from-javascript-to-python.mdx [hunk 2]
```diff
@@ -0,0 +1,249 @@
...
+- using the `@classmethod` decorator
+- taking time to learn about common libraries
+- writing simpler functions
+- being a bit lazier
+
+I'm still far from an expert but I hope these tips help you to avoid many of the mistakes that I made when starting out with Python.
+
+## use the `@classmethod` decorator
+
+I think more people should be thinking about classmethods and using the `@classmethod` decorator. They make your code a lot easier to mantain and reduce a chance of some complex initialisation logic backfiring on you.
```
Comment (jxnl) [inline posts/from-javascript-to-python.mdx:22]: Could be nice to sight the "dont use this code" guy

File: posts/from-javascript-to-python.mdx [hunk 3]
```diff
@@ -0,0 +1,249 @@
...
+class EmbeddingModel:
+    def __init__(
+        self,
+        model_name: str,
+        provider: Provider,
+        max_limit: int = 20,
+        expected_iterations: int = float("inf"),
+    ):
+      self.model_name = model_name
+      self.provider=provider
```
Comment (jxnl) [inline posts/from-javascript-to-python.mdx:49]: formatting

File: posts/from-javascript-to-python.mdx [hunk 4]
```diff
@@ -0,0 +1,249 @@
...
+import pandas as pd
+
+data = [{"key1": 2, "key4": 10}, {"key3": 3}, {"key4": 4}]
+df = pd.DataFrame(data)
+df.to_csv("output.csv", index=False)
+```
+
+That's a huge reduction in the potential issues with my code because I'm now using a library method that other people have spent time and effort to write and test. Additionally, using a common library helps you catch edge cases that you might otherwise miss in your hand-rolled implementation.
+
+## Write Simpler Functions
```
Comment (jxnl) [inline posts/from-javascript-to-python.mdx:121]: love this

File: posts/from-javascript-to-python.mdx [hunk 5]
```diff
@@ -0,0 +1,249 @@
...
+
+## Being a bit lazier
+
+it's actually not a bad thing to use a slightly less abstract solution sometimes. For instance, when I implement functions, I tend to prefer returning simple dictionaries if the response object isn't being shared between different functions.
+
+```python
+def extract_attributes(data):
+  # Do some processing
+	new_data = process_data(data)
+  return {
```
Comment (jxnl) [inline posts/from-javascript-to-python.mdx:212]: formatting
